modules/library.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Kindly refrain from using `eslint-disable` as this defies the purpose of using linters in the first place.